core/testenvmanager/dataset/dataset.py

```diff
@@ -148,6 +173,12 @@ def split_dataset(self, dataset_url, dataset_format, ratio, method="default",
                                               data_types=dataset_types,
                                               output_dir=output_dir,
                                               times=times)
+        # add new splitting method for semantic segmantation
+        if method == "my_splitting":
+            return self._my_splitting(dataset_url, dataset_format, ratio,

```

Comment:
The function name "_my_splitting" is not explicit.

Reply:
Ok, I will change the name of the function